### PR TITLE
POL-161 NTLM and Credentials Services changes

### PR DESCRIPTION
**Pull request by nshivappa**
### Description

NTLM and Credentials Services changes.

### Issues Resolved

- Added support for on-premise FlexNet Manager (NTLM auth).
- Changes to support the Credential Service.

### Contribution Check List

- [ ] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### Running Template Link and Screenshot

**AWS EC2 Instances not running FlexNet Inventory Agent:**
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e189bb832401200016df0f3
image

**Azure Instances not running FlexNet Inventory Agent:**
https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5e170b027abe69000112a92f
image

**FlexNet Manager Licenses At Risk:**
https://governance.rightscale.com/org/1105/projects/60073/applied-policies/5e181d15513cd00001eaec4b

### CODE REVIEW

**Comment by nshivappa** [comment]
For applying policy I have mentioned credentials type for NTLM as 'basic',
I will change it when bug(https://jira.flexera.com/browse/PHX-1973) gets resolved.

File: operational/fnms/schedule_fnms_reports_on_premise/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## v1.5
+
+- Added support for on-premise FlexNet Manager
```
Comment (cdwilhelm) [inline operational/fnms/schedule_fnms_reports_on_premise/CHANGELOG.md]: Should be version 2.0

File: operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt [hunk 2]
```diff
@@ -21,6 +26,7 @@ parameter "param_fnms_url" do
   label "FNMS Report URL"
   default "https://flexera-demo-1.flexnetmanager.com/Suite"
```
[the remark this answers is not in the document]
Comment (cdwilhelm) [reply operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt]: Same thing on the param_report_id

File: operational/fnms/schedule_fnms_reports_on_premise/CHANGELOG.md (hunk 1 above)
Comment (nshivappa) [reply operational/fnms/schedule_fnms_reports_on_premise/CHANGELOG.md]: Changed it to version 2.0

File: operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt (hunk 2 above)
[the remark this answers is not in the document]
Comment (nshivappa) [reply operational/fnms/schedule_fnms_reports/schedule-fnms-report.pt]: Updated, thanks

**Comment by nshivappa** [comment]
@cdwilhelm 
There is an ERROR saying "Invalid Danger file".
https://github.com/flexera/policy_templates/pull/431#issuecomment-573128744

**Comment by cdwilhelm** [comment]
@nshivappa  sorry, I missed the Dangerfile issues.  There are some broken urls and other danger errors.  can you please resolved these.  thanks.  Almost there.

File: compliance/aws/instances_without_fnm_agent/aws_instances_not_running_flexnet_inventory_agent.pt [hunk 3]
```diff
@@ -54,18 +67,19 @@ end
...
 ###############################################################################
 
-auth "auth_aws", type: "aws" do
-  version "4"
-  service "ec2"
-  access_key cred("AWS_ACCESS_KEY_ID")
-  secret_key cred("AWS_SECRET_ACCESS_KEY")
+#authenticate with AWS
+credentials "auth_aws" do
+  schemes "aws"
```
Comment (cdwilhelm) [inline compliance/aws/instances_without_fnm_agent/aws_instances_not_running_flexnet_inventory_agent.pt]: Sorry, we found schemes for AWS policies need "aws", "aws_sts"

**Pull request merged**